src/test/java/utilities/Util.java

```diff
@@ -1,7 +1,16 @@
 package utilities;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
 public class Util {
 
+	
+	
+	
+	public static void commandClearFunction(WebDriver driver) {
+		driver.findElement(By.id("UserName")).clear();
```

Comment:
Hello! I think you should add a String parameter for UserName so it can be reusable to other websites too :)

Reply:
it might be also easier to make this an xPath since you can always work with that on any website vs. searching for the element by id